### PR TITLE
[stable/gocd] Move the GoCD Helm Chart to stable

### PR DESCRIPTION
**Pull request by varshavaradarajan**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by varshavaradarajan** [comment]
@unguiculus - We are planning to announce the GoCD Helm chart on March 16th. Do you think its possible to get this merged by then?

File: stable/gocd/templates/gocd-ea-cluster-role-binding.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{{ if .Values.rbac.create }}
+apiVersion: rbac.authorization.k8s.io/{{ required "A valid .Values.rbac.apiVersion entry required!" .Values.rbac.apiVersion }}
+kind: ClusterRoleBinding
+metadata:
+  name: {{ template "gocd.fullname" . }}
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
```
Comment (unguiculus) [inline stable/gocd/templates/gocd-ea-cluster-role-binding.yaml:7]: app: "{{ template "gocd.name" . }}"

File: stable/gocd/templates/gocd-ea-cluster-role.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{{ if and .Values.rbac.create (not .Values.rbac.roleRef) }}
+apiVersion: rbac.authorization.k8s.io/{{ required "A valid .Values.rbac.apiVersion entry required!" .Values.rbac.apiVersion }}
+kind: ClusterRole
+metadata:
+  name: {{ template "gocd.fullname" . }}
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    app: "{{ template "gocd.fullname" . }}"
```
Comment (unguiculus) [inline stable/gocd/templates/gocd-ea-cluster-role.yaml]: app: "{{ template "gocd.name" . }}

File: stable/gocd/templates/gocd-ea-service-account.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{{ if .Values.serviceAccount.create }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ template "gocd.serviceAccountName" . }}
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    app: "{{ template "gocd.fullname" . }}"
```
Comment (unguiculus) [inline stable/gocd/templates/gocd-ea-service-account.yaml]: app: "{{ template "gocd.name" . }}"

**Comment by unguiculus** [comment]
@varshavaradarajan Sure, we should be able to make that.

I'd be great if you could update `_helpers.tpl` with the `fullname` template you get with `helm create` using Helm 2.8+.

**Comment by varshavaradarajan** [comment]
@unguiculus - thank you for that response. :) I have updated the PR with the helpers.tpl.

**Pull request merged**